Articles/WhatIs/WhatAreConferencesAndWorkshops.md

```diff
@@ -0,0 +1,28 @@
+### What are Conferences and Workshops for Better Scientific Software?
+#### Contributed by  [Rinku Gupta](https://github.com/rinkug)
+#### Publication date: April 20, 2023
+
+<!--- deck start --->
+Empowering researchers in scientifivc computing with knowledge-sharing, collaboration, and networking opportunities through conferences and workshops.
+<!--- deck end --->
+
+<!--- body start --->
+
+Conferences and workshops play a pivotal role in advancing scientific research in the field of scientific computing. These events offer researchers a valuable platform to showcase their latest findings, discuss and exchange ideas with other experts, and cultivate collaborations that drive innovation in their field. Beyond knowledge-sharing and collaboration, attending conferences and workshops enables researchers to remain up-to-date on the latest trends and advancements in their field, enabling them to refine their research approach and stay ahead of the curve.
```

Comment:
Do they *advance* the research itself or primarily serve as a forum for *communicating* the research? I think the latter.

Reply:
@rinkug As an older but newer researcher, I think you should try to get rid of the term younger and replace it with new researcher or somethink like a researcher new to the field, or emerging researcher (so no ageism)

Reply:
sounds good!